### PR TITLE
fix(curriculum): use example in Modifying an Object Within a Nested Object challenge

### LINKED ISSUE #38463: Remove redundant challenge (or fix it)
Challenge: https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-data-structures/add-key-value-pairs-to-javascript-objects

The challenge is part of the Basic Data Structures. It seems completely redundant, not to mention incorrectly implemented.

1. It is redundant because we already have [a challenge](https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-javascript/add-new-properties-to-a-javascript-object) in the basic javascript part of the curriculum that is exactly the same (except it actually shows the syntax).

2. It is incorrectly implemented because it does not show the syntax for setting key/values, it just shows how to get them. Which is not the requirement of the challenge.

I suggest we remove this challenge. Or if nothing else, correct it to show the syntax for setting key/values.

Forum report: https://www.freecodecamp.org/forum/t/dont-need-help-more-reporting/364888

### LINKED ISSUE #38467: Example could be added to explain to users how to modify an object within a nested object
**Describe your problem and how to reproduce it:**
Follow up from issue #38463

**Add a Link to the page with the problem:**

https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-data-structures/modify-an-object-nested-within-an-object

### PR DESCRIPTION
**Pull request by AshNaz87**
…se to explain how to do so

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #38463

### CODE REVIEW

**Comment by AshNaz87** [comment]
Screenshot 2020-03-29 at 21 59 29

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-data-structures/modify-an-object-nested-within-an-object.english.md [hunk 1]
```diff
@@ -18,18 +18,24 @@ let nestedObject = {
...
-      away: 13
+      away: 13,
+      busy: 8
     }
   }
 };
 ```
 
-<code>nestedObject</code> has three unique keys: <code>id</code>, whose value is a number, <code>date</code> whose value is a string, and <code>data</code>, whose value is an object which has yet another object nested within it. While structures can quickly become complex, we can still use the same notations to access the information we need.
+<code>nestedObject</code> has three unique keys: <code>id</code>, whose value is a number, <code>date</code> whose value is a string, and <code>data</code>, whose value is an object which has yet another object nested within it. While structures can quickly become complex, we can still use the same notations to access the information we need. To set the value of the <code>busy</code> key to <code>10</code>, we can do so using dot notation by doing:
```
Comment (RandellDawson) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-data-structures/modify-an-object-nested-within-an-object.english.md]:
```suggestion
<code>nestedObject</code> has three properties: <code>id</code> (value is a number), <code>date</code>(value is a string), and <code>data</code> (value is an object with its nested structure). While structures can quickly become complex, we can still use the same notations to access the information we need. To assign the value <code>10</code> to the <code>busy</code> property of the nested <code>onlineStatus</code> object, we use dot notation to reference the property:
```

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-data-structures/modify-an-object-nested-within-an-object.english.md [hunk 2]
```diff
@@ -18,18 +18,24 @@ let nestedObject = {
...
+```js
+nestedObject.data.onlineStatus.busy = 10;
+```
+
 </section>
 
 ## Instructions
 <section id='instructions'>
-Here we've defined an object, <code>userActivity</code>, which includes another object nested within it. You can modify properties on this nested object in the same way you modified properties in the last challenge. Set the value of the <code>online</code> key to <code>45</code>.
+Here we've defined an object, <code>userActivity</code>, which includes another object nested within it. Set the value of the <code>online</code> key to <code>45</code>.
```
Comment (RandellDawson) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-data-structures/modify-an-object-nested-within-an-object.english.md]:
```suggestion
Here we've defined an object <code>userActivity</code> that includes another object nested within it. Set the value of the <code>online</code> key to <code>45</code>.
```

**Review by RandellDawson — DISMISSED** [review]
See my suggestions for making the wording a bit more concise.

**Comment by AshNaz87** [comment]
@RandellDawson Is this good to go?

**Review by moT01 — APPROVED** [review]
LGTM @AshNaz87 🎉

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-data-structures/modify-an-object-nested-within-an-object.english.md [hunk 3]
```diff
@@ -18,18 +18,24 @@ let nestedObject = {
...
-      away: 13
+      away: 13,
+      busy: 8
     }
   }
 };
 ```
 
-<code>nestedObject</code> has three unique keys: <code>id</code>, whose value is a number, <code>date</code> whose value is a string, and <code>data</code>, whose value is an object which has yet another object nested within it. While structures can quickly become complex, we can still use the same notations to access the information we need.
+<code>nestedObject</code> has three properties: <code>id</code> (value is a number), <code>date</code>(value is a string), and <code>data</code> (value is an object with its nested structure). While structures can quickly become complex, we can still use the same notations to access the information we need. To assign the value <code>10</code> to the <code>busy</code> property of the nested <code>onlineStatus</code> object, we use dot notation to reference the property:
```
Comment (Sky020) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-data-structures/modify-an-object-nested-within-an-object.english.md:28]:
```suggestion
<code>nestedObject</code> has three properties: <code>id</code> (value is a number), <code>date</code> (value is a string), and <code>data</code> (value is an object with its nested structure). While structures can quickly become complex, we can still use the same notations to access the information we need. To assign the value <code>10</code> to the <code>busy</code> property of the nested <code>onlineStatus</code> object, we use dot notation to reference the property:
```

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-data-structures/modify-an-object-nested-within-an-object.english.md [hunk 4]
```diff
@@ -18,18 +18,24 @@ let nestedObject = {
...
+```js
+nestedObject.data.onlineStatus.busy = 10;
+```
+
 </section>
 
 ## Instructions
 <section id='instructions'>
-Here we've defined an object, <code>userActivity</code>, which includes another object nested within it. You can modify properties on this nested object in the same way you modified properties in the last challenge. Set the value of the <code>online</code> key to <code>45</code>.
+Here we've defined an object <code>userActivity</code> that includes another object nested within it. Set the value of the <code>online</code> key to <code>45</code>.
```
Comment (Sky020) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-data-structures/modify-an-object-nested-within-an-object.english.md:38]:
```suggestion
Here we've defined an object <code>userActivity</code>, which includes another object nested within it. Set the value of the <code>online</code> key to <code>45</code>.
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
Just some nitpicking.

**Review by ShaunSHamilton — APPROVED** [review]
LGTM 👍

**Comment by moT01** [comment]
Lets try again...

**Review by moT01 — APPROVED** [review]
LGTM @AshNaz87 🎉 Thanks for contributing to freeCodeCamp 🎉

**Pull request merged**